seps/0000-dpop-extension.md

```diff
@@ -0,0 +1,165 @@
+# SEP-1932: DPoP Profile for MCP
+
+> **Note**: This SEP defines an optional security extension for MCP that enables sender-constrained access tokens through DPoP (Demonstrating Proof-of-Possession).
+
+- **Status**: Draft
+- **Type**: Standards Track
+- **Created**: 2025-12-05
+- **Author(s)**: Pieter Kasselman
+- **Sponsor**: Darin McAdams
+- **PR**: https://github.com/modelcontextprotocol/ext-auth/blob/pieterkas-dpop-extension/specification/draft/dpop-extension.mdx
+
+## Abstract
+
+This SEP defines an optional DPoP (Demonstrating Proof-of-Possession) extension for the Model Context Protocol that enables sender-constrained access tokens to prevent token replay attacks. The extension binds OAuth 2.0 access tokens to cryptographic key pairs controlled by MCP clients, requiring clients to prove possession of the private key with each request. A key enhancement specific to MCP's architecture is the required inclusion of a content digest in DPoP proofs, which cryptographically binds each proof to the specific JSON-RPC request body, addressing security challenges posed by MCP's single-endpoint architecture.
```

Comment:
```suggestion
This SEP defines an optional DPoP (Demonstrating Proof of Possession) extension for the Model Context Protocol that enables sender-constrained access tokens to prevent token replay attacks. The extension binds OAuth 2.0 access tokens to cryptographic key pairs controlled by MCP clients, requiring clients to prove possession of the private key with each request. A key enhancement specific to MCP's architecture is the required inclusion of a content digest in DPoP proofs, which cryptographically binds each proof to the specific JSON-RPC request body, addressing security challenges posed by MCP's single-endpoint architecture.
```